.github/workflows/integration.yml

```diff
@@ -46,25 +46,10 @@ jobs:
       int_tests: true
     secrets: inherit
 
-  detect-user:
```

Comment:
An alternative here would be to let this fail but then mark the user as not a team member if it does.

Reply:
I considered that too ... I guess it has similar drawbacks re false positives as the secrets detection approach . Like if the job failed for a different reason, we might skip these int tests when we shouldn't be.

Reply:
And also we then wouldn't be able to consider this job's status at the end to determine whether the whole workflow should be considered as failed/success

Reply:
but in the "happy path" of it failing, it's failing due to the secret not being defined, so the current method I have is more precisely looking at that.

Reply:
Mmm, yeah, that makes sense. Maybe we could call the output `PR_HAS_ACCESS_TO_SECRETS`? That seems like a more precise name.